hedging/tests/hedging.rs

```diff
@@ -138,3 +139,69 @@ async fn hedging() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+#[serial]
+async fn scenario_01() -> anyhow::Result<()> {
+    let redis_host = std::env::var("REDIS_HOST").unwrap_or("localhost".to_string());
+    let pubsub_config = PubSubConfig {
+        host: Some(redis_host),
+        ..PubSubConfig::default()
+    };
+    let user_trades_pg_host = std::env::var("HEDGING_PG_HOST").unwrap_or("localhost".to_string());
+    let user_trades_pg_port = std::env::var("HEDGING_PG_PORT").unwrap_or("5433".to_string());
+    let pg_con = format!(
+        "postgres://stablesats:stablesats@{user_trades_pg_host}:{user_trades_pg_port}/stablesats-hedging"
+    );
+
+    let publisher = Publisher::new(pubsub_config.clone()).await?;
+    let subscriber = Subscriber::new(pubsub_config.clone()).await?;
+    let mut stream = subscriber
+        .subscribe::<OkexBtcUsdSwapPositionPayload>()
+        .await?;
+
+    tokio::spawn(async move {
+        HedgingApp::run(
+            HedgingAppConfig {
+                pg_con,
+                migrate_on_start: true,
+                okex_poll_frequency: std::time::Duration::from_secs(1),
+            },
+            okex_client_config(),
+            pubsub_config,
+        )
+        .await
+    });
+    tokio::time::sleep(std::time::Duration::from_secs(3)).await;
+
+    let expected_exposure = [
```

Comment:
I don't like that this is defined here while the fixtures are in the json blob.
Can we either derive the expectation from the payload or encode them together?
Perhaps:
scenario_01.json
```
{
  scenario: [
  { payload: { liability: 100}
    expectation: { lowerBound: 99, upperBound: 101 }
    },
    <etc>
  ]
}
```

Reply:
Yeah wanted to do exactly that, and was going to ask what you thought was the best way...

Reply:
I think the best way would be to derive the expectation from the `payload.liability`. If the upper / lower bound is always +/- 1 this should be simple.